data.sql

```diff
@@ -162,6 +162,43 @@ INSERT INTO animals(
     17
 );
 
+-- Inserting data into owners table
+INSERT INTO owners(owner_id,full_name,age) VALUES (1,'sam smith',34);
+INSERT INTO owners(owner_id,full_name,age) VALUES (2,'Jennifer Orwell',19);
+INSERT INTO owners(owner_id,full_name,age) VALUES (3,'Bob',45);
+INSERT INTO owners(owner_id,full_name,age) VALUES (4,'Melody Pond',77);
+INSERT INTO owners(owner_id,full_name,age) VALUES (5,'Dean Winchester',14);
+INSERT INTO owners(owner_id,full_name,age) VALUES (6,'Jodie Whittaker',38);
+
+-- Inserting daat into Species table
+INSERT INTO species(species_id,name) VALUES (1,'pokemon');
+INSERT INTO species(species_id,name) VALUES (2,'Digimon'
+
+-- Modify your inserted animals to include owner information (owner_id):
```

Comment:
- [ ] There is currently no query for modifying your inserted animals so it includes the species_id value: if the name ends in "mon" it will be Digimon. e.g. 
```sql
UPDATE animals
SET species_id = (SELECT id from species WHERE name = 'Digimon')
WHERE name like '%mon';
```